configure.ac

```diff
@@ -999,12 +1000,16 @@ else
     WELCOME_CONFIG_FRAGMENT="<welcome>
       <enable type=\"bool\" desc=\"Controls whether the welcome screen should be shown to the users on new install and updates. When enabled, your custom welcome screen must be created at /usr/share/coolwsd/browser/dist/welcome/welcome.html.\" default=\"false\">false</enable>
     </welcome>"
+    CONFIG_INTERFACE_FRAGMENT="<brandProductName desc=\"Change the displayed name the application will have for the user.\" type=\"string\"></brandProductName>
+    <brandProductURL desck=\"Change the url of the , empty makes disable the link\" type=\"string\">https://www.collaboraonline.com</brandProductURL>"
```

Comment:
typo: `desck`
Incomplete sentence in description

Reply:
> It will not work with brand package, because settings in branding.js will overwrite the settings in global.js.

It is meant to be used when no branding is set.

> branding_override.js, and load it after branding.js

I'd like to avoid adding a new request, plus this file has to be dynamic since it depends on settings, so that would adding some file generation logic and serving. It seems to me way too complicated for what the feature I am adding.

Having a different name with Override postfix would make things compatible, but do we really need it ?